### ISSUE TITLE
Migrate to HASS.IO

### ISSUE THREAD

**Issue by CCOSTAN** [CCOSTAN/Home-AssistantConfig#233]
Move to new SD Card and begin migration and testing.

**Comment by CCOSTAN** [comment]
Move to intel NUC.  need to investigate hardware requirements for VMware install as well.

**Comment by Amir974** [comment]
If you have any input on how to migrate packages and scripts to hassio - a step by step guide would be awesome...

from the beginner post it seems the process involves the creation of a personal addon (not sure why there is no blank one already setup), 
I am trying to figure out from the addon tutorial process what exactly is required, I mean - that add on is not intended to run a custom component - just a basic extension of the configuration files, so what exactly would it take ?

**Comment by evaryont** [comment]
I've gotten Hass.io running on an Arch linux VM with Docker, all automated with Ansible. I can share the role if you're interested in it -- it's meant for installing on generic systems, not using one of the images provided by the H-A team.

**Comment by CCOSTAN** [comment]
Thanks for the offer @evaryont.  I'm still waiting on the NUC hardware.  That's pending.. once I have that, I'll start the migration plans.  (AND write up @Amir974 )

Thanks
CARLO.

**Comment by mattthhdp** [comment]
Pretty much my setup ^^ i have an intel Nuc with Esxi on it, i install a ubuntu server vm, installed docker and finally home assistant !

**Comment by CCOSTAN** [comment]
Gonna stick with the Pi.  I feel like it was BUILT for HA.

**Comment by CCOSTAN** [comment]
@frenck has opened my eyes and put me on the right track again.  Time to try out the migration. 
https://gist.github.com/frenck/32b4f74919ca6b95b30c66f85976ec58

**Comment by CCOSTAN** [comment]
Tools to remove : 
Simply do the following:

stop services
(thanks @IcyMidnight)

sudo systemctl stop hassio-supervisor.service
sudo systemctl stop hassio-apparmor.service

disable services
sudo systemctl disable hassio-supervisor.service
sudo systemctl disable hassio-apparmor.service

remove services
sudo rm -rf /etc/systemd/system/hassio-supervisor.service
sudo rm -rf /etc/systemd/system/hassio-apparmor.service

removing hassio folders (except the config folder)
sudo rm -rf /usr/sbin/hassio-supervisor
sudo rm -rf /usr/sbin/hassio-apparmor

Now delete the remaining docker images like you always do.
.
.
.
.

If you want to delete your Home Assistant config files,
Do the following, but be carefull! Your config is lost forever:
sudo rm -rf /usr/share/hassio/

**Comment by pinkywafer** [comment]
Are you still on the pi?

**Comment by CCOSTAN** [comment]
No .. an old laptop.  Running Home Assistant Core.  Just going through all my old issues and closing out things that are no longer relevant.   (Because this conference call is BOOOOOOOOOORING!). :)
